README.md

```diff
@@ -1,2 +1,6399 @@
+                    </li>
+                </ul>
+            </td>
+        </tr>
+        <tr>
+            <td>Attributes in Result</td>
+            <td>(Optional) Specify a string array to limit the output by returning only the specified attributes. For example, <code>[&quot;comment&quot;, &quot;extended-log&quot;, &quot;feature-set&quot;, &quot;https-replacemsg&quot;, &quot;log-all-url&quot;, &quot;name&quot;, &quot;options&quot;, &quot;ovrd-perm&quot;, &quot;post-action&quot;, &quot;replacemsg-group&quot;, &quot;web-antiphishing-log&quot;, &quot;web-content-log&quot;, &quot;web-extended-all-action-log&quot;, &quot;web-filter-activex-log&quot;, &quot;web-filter-applet-log&quot;, &quot;web-filter-command-block-log&quot;, &quot;web-filter-cookie-log&quot;, &quot;web-filter-cookie-removal-log&quot;, &quot;web-filter-js-log&quot;, &quot;web-filter-jscript-log&quot;, &quot;web-filter-referer-log&quot;, &quot;web-filter-unknown-log&quot;, &quot;web-filter-vbs-log&quot;, &quot;web-ftgd-err-log&quot;, &quot;web-ftgd-quota-usage&quot;, &quot;web-invalid-domain-log&quot;, &quot;web-url-log&quot;, &quot;wisp&quot;, &quot;wisp-algorithm&quot;, &quot;wisp-servers&quot;]</code>. If attributes are not specified, then all attributes will be returned.</td>
+        </tr>
+        <tr>
+            <td>Filter By</td>
+            <td>(Optional) You can filter the result according to a set of criteria by specifying attributes in the format <code>[[&quot;&quot;, &quot;==&quot;, &quot;&quot;]]</code></td>
+        </tr>
+        <tr>
+            <td>Limit</td>
+            <td>(Optional) Specify the maximum number of results that this operation should return.</td>
+        </tr>
+        <tr>
+            <td>Offset</td>
+            <td>(Optional) Specify the offset value to retrieve a subset of records that starts from the offset value. The offset works with the <em>Limit</em> parameter, which determines how many records to retrieve starting from the offset. Values supported are:
+                <ul>
+                    <li><strong>Default:<code>&quot;0&quot;</code></strong></li>
+                    <li><strong>Minimum:<code>&quot;0&quot;</code></strong></li>
+                </ul>
+            </td>
+        </tr>
+        <tr>
+            <td>Sort By</td>
+            <td>Sort the web filters by a field and order the results. You can choose to either sort the results by fields, or can order the results, or both.&nbsp;
+                <ul>
+                    <li>If you choose &quot;Field&quot;, then in the <strong>Field</strong> field specify the name of the field on which you want to sort the result. Fields based on which you can sort are <code>account-key-filter, account-key-processing, antiphish, ca-cert, cnid, dn, group-filter, group-member-check, group-object-filter, group-search-base, interface, interface-select-method, member-attr, name, obtain-user-info, password, password-attr, password-expiry-warning, password-renewal, 
```

Comment:
please replace this line with "Select the level type from which you want to unblock the applications associated with the specified application control profile. You can choose between ADOM or Global Type."

Reply:
Done